ui/src/dashboards/containers/DashboardPage.tsx

```diff
@@ -372,21 +361,8 @@ class DashboardPage extends Component<Props, State> {
 
   private get showDashboard(): boolean {
     const {dashboard} = this.props
-    const {servicesStatus} = this.state
-
-    const showDashboard = dashboard && servicesStatus === RemoteDataState.Done
```

Comment:
what's the reasoning behind getting rid of the loading state check?

Reply:
its checking whether specifically services are loaded, but we are no longer loading services

Reply:
gotcha